src/platform/packages/shared/response-ops/alerts-table/components/empty_state.tsx

```diff
@@ -25,53 +26,77 @@ const panelStyle = {
 };
 
 export const EmptyState: React.FC<{
-  height?: keyof typeof heights;
+  height?: keyof typeof heights | 'flex';
```

Comment:
Added a flex height variant for the empty state to center it in the embeddable panel

Reply:
Before: 
<img width="601" alt="image" src="https://github.com/user-attachments/assets/e53de0c3-99c4-47ed-8506-68e3f4618f3d" />

After:
<img width="598" alt="image" src="https://github.com/user-attachments/assets/20fbdf88-5254-43a3-bd6e-1754e9fe07d1" />